x-pack/packages/security-solution/features/src/product_features_keys.ts

```diff
@@ -121,7 +126,7 @@ export enum SecuritySubFeatureId {
   endpointList = 'endpointListSubFeature',
   endpointExceptions = 'endpointExceptionsSubFeature',
   trustedApplications = 'trustedApplicationsSubFeature',
-  hostIsolationExceptions = 'hostIsolationExceptionsSubFeature',
+  hostIsolationExceptionsBasic = 'hostIsolationExceptionsSubFeature',
```

Comment:
```suggestion
  hostIsolationExceptionsBasic = 'hostIsolationExceptionsBasicSubFeature',
```

Reply:
Changed in 701474a6f447d0f5270de971873d5133edece4fb